verification/curator-service/src/storage/store.ts

```diff
@@ -0,0 +1,26 @@
+import { Db } from 'mongodb';
+
+const SOURCES_COLLECTION = 'sources';
+
+export interface Store {
+    listSources(): Promise<Array<object>>;
+}
+
+export class MongoStore implements Store {
+    constructor(private readonly db: Db) {}
+    async listSources(): Promise<Array<object>> {
+        const col = this.db.collection(SOURCES_COLLECTION);
+        return await col.find().toArray();
+    }
+}
+
+export class MemStore implements Store {
```

Comment:
For testing, there's a jest-mongodb package that simplifies directing MongoDB to an in-memory server ([jest docs](https://jestjs.io/docs/en/mongodb)). I'm not sure this set up is worth having to conditionally configure the implementation in our app, and writing the in-memory methods for each method/source. Mongoose offers this same object-orientation; we'd just operate on the model schemas in lieu of defining .*Store classes ([docs](https://www.npmjs.com/package/mongoose#accessing-a-model)).

My impression is that the default jest/mongoose set up will be simpler, but I'm interested in your take on the above.